src/theme/colors.ts

```diff
@@ -0,0 +1,65 @@
+// TODO: Replace with our real default colors
```

Comment:
This looks great!  Great thoughts offline here.  It's important (to me) that this lib represents the "LifeOmic mobile app shell."  It should _not_ end up being the new Outcomes repo.  So ideally, these colors/spacing/etc. represent sensible defaults for us to develop components against.  And then we're overriding those in our Outcomes repo.

Reply:
I'm curious whether this folder should `defaultTheme`, `baseTheme` - something to that effect?  Esp. if it's wireframe-y, I'd think we always want/expect something layered on top of the default values provided in these files.